shared/macros-bash.jinja

```diff
@@ -71,7 +71,7 @@ printf '%s\n' '{{{ message }}}' >&2
 # separator_regex: regular expression that describes the separator and surrounding whitespace
 # prefix_regex: regular expression describing allowed leading characters at each line
 #}}
-{{%- macro set_config_file(path, parameter, value, create, insert_after, insert_before, insensitive, separator, separator_regex, prefix_regex) -%}}
+{{%- macro set_config_file(path, parameter, value, create, insert_after, insert_before, insensitive=True, separator=" ", separator_regex="\s\+", prefix_regex="^\s*") -%}}
```

Comment:
Have we think about insensitive should be default True or False?

Reply:
I would leave it like it is. Tests will assure that it won't flip unnoticed.